source/NVDAObjects/UIA/edge.py

```diff
@@ -516,6 +517,16 @@ def _get_landmark(self):
 				return ariaRole
 		return None
 
+	def event_liveRegionChange(self):
+		# #8466: elements with aria-role=alert set fires live region changed event but does not expose the alert text as its name, wihch can be found in its descendants.
+		if self.ariaProperties.get("role") == "alert" and self.treeInterceptor:

```

Comment:
Wouldn't this make live region events go silent if this predicate is not True? I'd say you have to check that the object has no name as well. furthermore, if the predicate is False, please call the super method.